docs/schema/aws.md

```diff
@@ -24,66 +24,70 @@
   - [Relationships](#relationships-7)
 - [AWSPrincipal::AWSRole](#awsprincipalawsrole)
   - [Relationships](#relationships-8)
-- [AWSVpc](#awsvpc)
+- [AWSTransitGateway](#awstransitgateway)
```

Comment:
The doc seemed to be in alphabetical order, so I tried to keep within that order, but if you think it would be better to move it to the bottom I'll do that.